crates/bws/src/main.rs

```diff
@@ -97,6 +107,57 @@ enum Commands {
     },
 }
 
+#[derive(Subcommand, Debug)]
+enum SecretCommand {
+    Get {
+        secret_id: Uuid,
+    },
+    Create {
+        key: String,
+        value: String,
+
+        #[arg(long, help = "An optional note to add to the secret")]
+        note: Option<String>,
+
+        #[arg(long, help = "The ID of the project this secret will be added to")]
+        project_id: Option<Uuid>,
+    },
+    List {
+        project_id: Option<Uuid>,
+    },
```

Comment:
I think we should move List to the top, or bottom. The other commands works on a single entity so in my opinion it makes sense to group them. Or group everything alphabetically.

Reply:
In case we add other commands in the future that may be unrelated and harder to sort... I have sorted the enum variants in `bws/src/main.rs` alphabetically.